array_api_compat/__init__.py

```diff
@@ -17,6 +17,6 @@
 this implementation for the default when working with NumPy arrays.
 
 """
-__version__ = '1.9.1'
+__version__ = '1.10.0dev0'
```

Comment:
Needed to work around https://github.com/prefix-dev/pixi/issues/2677
Without this we can't build array-api-extra until the next release: https://github.com/data-apis/array-api-extra/pull/53#discussion_r1876022501